environment.yml

```diff
@@ -27,8 +27,7 @@ dependencies:
 
   # documentation
   - gitpython  # obtain contributors from git for whatsnew
-  # some styling is broken with sphinx >= 2 (https://github.com/pandas-dev/pandas/issues/26058)
-  - sphinx=1.8.5
+  - sphinx
```

Comment:
should we have a min sphinx now, e.g. sphinx>=2.0.0? (also do we need to update contributing.rst?)

Reply:
It's not that the the docs are not building with the older sphinx version, so it's not a hard requirement (there is only some javascript based functionality that is not working, but which is not important for people contributing to the docstrings)

In the contributing guide we don't mention anything about the sphinx version currently.